dll/win32/wdmaud.drv/mmixer.c

```diff
@@ -391,7 +391,7 @@ WdmAudCleanupByMMixer()
 }
 
 MMRESULT
-WdmAudGetMixerCapabilities(
+WdmAudGetMixerCapabilties(
```

Comment:
your commit introduced a typo in that function name. Bring back the old name. But be warned: You have to fix this at more than just this one place or the code will not longer compile. Ideally try fixing this within a single commit, where you do nothing else at the same time. Use the compiler to verify your success!

Reply:
Fixed.